tests/test_core_utils_coverage.py

```
@@ -28,6 +29,19 @@ def setUp(self):
             "matrix_rooms": [{"id": "!room1:example.com", "meshtastic_channel": 0}],
         }
 
+    @staticmethod
+    def _immediate_metadata_submit(callback):
```

Ruff (0.15.10)

[warning] Missing return type annotation for staticmethod `_immediate_metadata_submit`

(ANN205)